resolvers/Mutation.js

```diff
@@ -17,11 +17,21 @@ const Mutation = {
     async createItem(parent, {data}, {}, info){
         console.log("recevied data:", data)
         let itemId = "item"+Math.floor(Math.random()*1000)
-        let result = await dbMutation(`INSERT INTO \`Item\` VALUES('${itemId}', '${data.img}', ${data.price} )`)
+        try{
+            let sql = `INSERT INTO \`Item\` VALUES('${itemId}', '${data.img}', ${data.price} )`
+            let result = await dbMutation(sql)
+        }catch(e){
+            console.log("Fail sql: ", sql, e)
```

Comment:
so by wrapping it with **try catch block** you made it so when the error occurs we'll just not going to show it to the frontend but we'll actually fail to insert the data into the database?

Reply:
Yes, if the error occurs, we fail the corresponding sql query.

Reply:
but why is the sql query failing in the first place? In theory mysql should be powerful enough to easily handle thousands of INSERT requests per second

Reply:
You are right. In fact, I solve the issue we discuss in Line by adds close connection command. I guess that's why the issue is caused.